lib/drivers/veres/index.js

```diff
@@ -21,6 +25,78 @@ const _keyTypes = {
   rsa: 'RsaVerificationKey2018'
 };
 
+function driver({mode, hostname}) {
+  // TODO: Support custom httpsAgent?
+  const httpsAgent = new https.Agent({rejectUnauthorized: mode !== 'dev'});
+
+  return v1.driver({mode, hostname, httpsAgent});
+}
+
+/**
+ * @param {string} [type='veres'] - DID type / method.
+ * @param {string} [mode='test'] - Ledger mode (test/live/dev).
+ *
+ * @returns {{pending: FlexDocStore, registered: FlexDocStore}}
+ */
+function didStoreFor({type, mode}) {
```

Comment:
the jsdoc indicate that type and mode are defaulted, but I don't see that in the params.

Reply:
The default is set back in the `did` binary, in yargs. The jsdoc here is just a reminder. If you'd like, I can remove that part.

Reply:
jsdoc is not usually used to indicate what values a caller is passing into a function is it?

If I want to consume this API and look at the jsdoc, I would get the impression that mode and type are optional and if not supplied, mode will default to 'test' and type will default to 'veres'.

https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
No prob, I made the defaults explicit, in the params declaration.